code/game/gamemodes/cult/runes.dm

```diff
@@ -1132,7 +1132,8 @@
 			if (dna_scannernew.locked)
 				dna_scannernew.locked = 0
 		for(var/mob/living/carbon/C in users)
-			C.take_overall_damage(10, 0)
+			var/rune_damage = 20 / (users.len)
```

Comment:
Move the variable declaration to before the for loop, else you're just redeclaring a variable needlessly.